static/js/student-tile.ts

```diff
@@ -0,0 +1,36 @@
+function expandStudentTile(student: any) {
+    const average_quiz = student.average_quiz;
+    const success_rate_highest_level = student.success_rate_highest_level;
+    const success_rate_overall = student.success_rate_overall;
+    const highest_level_quiz = student.highest_level_quiz;
+    const highest_level_quiz_score = student.highest_level_quiz_score;
+
+    let bigTile = document.getElementById('expanded-student-tile')!;
+    bigTile.innerHTML =
```

Comment:
This should be done with either HTMX or with a flask/jinja template and not directly in the typescript.

Reply:
issue fixed.

Reply:
Sorry Dvolpin! But I don't see where this issue was fixed? If you need help with HTMX or flask you can hit me up!

Reply:
I will try to fix to work on this tomorrow, if I get stuck on it, I will ping you! Thanks for the various inputs!